src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1612,5 +1630,9 @@ def pick_strategies(strategy_pool, fuzzer_path, corpus_directory,
       use_mutator_plugin(target_name, extra_env)):
     fuzzing_strategies.append(strategy.MUTATOR_PLUGIN_STRATEGY.name)
 
+  if (strategy_pool.do_strategy(strategy.MUTATOR_PLUGIN_RADAMSA_STRATEGY) and
+      use_radamsa_mutator_plugin(extra_env)):
+    fuzzing_strategies.append(strategy.MUTATOR_PLUGIN_RADAMSA_STRATEGY.name)
```

Comment:
you've mentioned earlier today that the mutator might not work for some reason. If that's a real possibility, we might want to have some check to make sure we don't record `'strategy_mutator_plugin_radamsa': 1` in stats when the mutator actually failed to get loaded

Reply:
Depending on how much we want to do this, I think printing on the first run of the mutation and checking for this print could be a good solution.

Reply:
Printing can be error prone, especially if we need every plugin to conform to some format. Maybe we can modify libFuzzer to add an argument for mutator plugins (and which checks that it loads properly).

Reply:
I have not had any success getting the libradamsa shared object to output to the log.

Reply:
I think a can make a change like this in libFuzzer. Shouldn't be blocking you right now, but let's have a TODO to address in future?

Reply:
Uploaded https://reviews.llvm.org/D73136 for review.

Reply:
@mpherman2 but don't wait on this, even if it gets landed, it'll take a couple weeks for the change to propagate to all ecosystems where ClusterFuzz is running.